horreum-backend/src/test/java/io/hyperfoil/tools/horreum/svc/BaseServiceTest.java

```diff
@@ -192,12 +193,13 @@ protected void dropAllViewsAndTests() {
 
                 em.createNativeQuery("DELETE FROM test_transformers").executeUpdate();
                 em.createNativeQuery("DELETE FROM transformer_extractors").executeUpdate();
+                em.createNativeQuery("DELETE FROM experiment_comparisons").executeUpdate();
```

Comment:
Does this relate to another issue?

Reply:
no, I get ` ERROR: update or delete on table "variable" violates foreign key constraint "experiment_comparison_variable" on table "experiment_comparisons"
  Detail: Key is still referenced from table "experiment_comparisons".` errors without it.